### ISSUE TITLE
block-closing-brace-newline-after does not flag when the space after brace

### ISSUE THREAD

**Issue by in-in** [stylelint/stylelint#1842]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug
`block-closing-brace-newline-after` does not flag when the space after brace

> Which rule, if any, is this issue related to?

`"block-closing-brace-newline-after": "always"`

> What CSS is needed to reproduce this issue?

```css
a { color: #000;
} b { color: #fff; }
```

or

```css
a { color: #000;
}b { color: #fff; }
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "block-closing-brace-newline-after": "always"
  }
}
```

> Which version of stylelint are you using?

stylelint@7.1.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

node scripts with PostCss

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

I expected warning

> What actually happened (e.g. what warnings or errors you are getting)?

No warning

**Comment by jeddy3** [comment]
@in-in I can't replicate this one I'm afraid. We also [have tests](https://github.com/stylelint/stylelint/blob/master/src/rules/block-closing-brace-newline-after/__tests__/index.js#L45-L55) for your two examples. 

Can you paste your config and code snippet into the [demo](http://stylelint.io/demo/) and try there. If you don't get any warnings can you create a PR with failing tests please, as it'd really help us investigate the issue.

**Comment by in-in** [comment]
Yes, you're right
Into demo it works ok
But same css and json do not work on my machine
Sorry for this false issue
